Source/Network/Router/MLCardFormApiRouter.swift

```diff
@@ -49,64 +49,60 @@ enum MLCardFormApiRouter {
-                    MLCardFormBinService.HeadersKeys.xFlowId.getKey: headers.xFlowId]
+                    MLCardFormBinService.HeadersKeys.xFlowId.getKey: headers.xFlowId,
+                    MLCardFormBinService.HeadersKeys.sessionId.getKey: headers.sessionId,
+                    MLCardFormWebPayService.HeadersKeys.accessToken.getKey: headers.accessToken]
         }
     }
 
     var parameters: [URLQueryItem] {
-        switch self {
-        case .getCardData(let queryParams, _):
-            var urlQueryItems = [
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.bin.getKey, value: queryParams.bin),
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.siteId.getKey, value: queryParams.siteId),
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.platform.getKey, value: queryParams.platform),
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.odr.getKey, value: String(queryParams.odr))
-            ]
-            if let excludedPaymentTypes = queryParams.excludedPaymentTypes {
-                urlQueryItems.append(URLQueryItem(name: MLCardFormBinService.QueryKeys.excludedPaymentTypes.getKey, value: excludedPaymentTypes))
-            }
-            return urlQueryItems
-        case.postCardTokenData(let queryParams, _, _):
-            var urlQueryItems:[URLQueryItem] = []
-            if let accessToken = queryParams.accessToken {
-                urlQueryItems.append(URLQueryItem(name: MLCardFormAddCardService.QueryKeys.accessToken.getKey, value: accessToken))
-            } else if let publicKey = queryParams.publicKey {
-                urlQueryItems.append(URLQueryItem(name: MLCardFormAddCardService.QueryKeys.publicKey.getKey, value: publicKey))
+            switch self {
+            case .getCardData(let queryParams, _):
+                var urlQueryItems = [
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.bin.getKey, value: queryParams.bin),
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.siteId.getKey, value: queryParams.siteId),
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.platform.getKey, value: queryParams.platform),
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.odr.getKey, value: String(queryParams.odr))
+                ]
+                if let excludedPaymentTypes = queryParams.excludedPaymentTypes {
+                    urlQueryItems.append(URLQueryItem(name: MLCardFormBinService.QueryKeys.excludedPaymentTypes.getKey, value: excludedPaymentTypes))
+                }
+                return urlQueryItems
+            case.postCardTokenData( _, _):
+                return [];
+            case .postCardData( _, _):
+                return [];
+            case .getWebPayInitInscription( _),
+                 .postWebPayFinishInscription( _, _):
+                return [];
+            case .getCardDataFromMarketplace(_):
```

Comment:
se esses caras todos retornam vazio, podia ser assim
```suggestion
            case.postCardTokenData, .postCardData, .getWebPayInitInscription, .postWebPayFinishInscription,  .getCardDataFromMarketplace:
```

Reply:
Sim, esse era um refactor que tinha visto para arrumar, vou fazer isso agora.
faz todo sentido....valeu

Reply:
feito